R/promptAndRun.R

```diff
@@ -11,6 +11,11 @@ promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
     loopRuns(if (length(chosendirs) == 0) "exit" else chosendirs, user = user)
   } else if (mydir == "-f") {
     loopRuns(dir(), user = user)
+  } else if (mydir == "-t") {
+    amtPath <- "/p/projects/remind/modeltests/output/"
+    amtPattern <- readRDS("/p/projects/remind/modeltests/runcode.rds")
+    amtDirs<-paste0(amtPath, dir(path = amtPath, pattern = amtPattern))
+    loopRuns(amtDirs, user = user)
```

Comment:
is the `user` argument correct like that, because it should be independent of a user, right? It works, though…

Reply:
yes, it is basically ignored in these cases